spec/api/grape_api/v1/repositories_spec.rb

```diff
@@ -86,8 +86,19 @@
       tags = JSON.parse(response.body)
       expect(response).to have_http_status(:success)
       expect(tags.length).to eq(2)
-      expect(tags[0].length).to eq(1)
-      expect(tags[1].length).to eq(4)
+
+      # The order depends on the update_at property, which might be slightly
+      # different depending on how fast creation happened on these tests.
```

Comment:
Really? Couldn't this be a side effect of the sort thing that you've changed? I dunno, just guessing.

Reply:
No, I did this before this other change 😉 Anyways, mysql was hitting the other case.

Reply:
Weird but ok. 👍